utp-testing/README.md

```diff
@@ -0,0 +1,7 @@
+# utp-testing
+
+Utp testing framework (WIP)
```

Comment:
Was there a reason for putting this in a new crate rather than `ethportal-peertest`?

Reply:
Hmm, the plan is to build uTP testing framework in the future with clent/server docker instances and network simulator between them as described [here.](https://github.com/marten-seemann/quic-network-simulator). Not sure how this is going to evolve exactly and if it makes sense to be part of `ethportal-peertest` at this point. Maybe we leave it as it is now in a separate crate and see?